### ISSUE TITLE
Import failed, file path in log is truncated; Sonarr mistakingly using the torrent name as folder path?

### ISSUE THREAD

**Issue by JustinReinhart** [Sonarr/Sonarr#4073]
**Describe the bug**
Bug: Import files, file is not copied. Log seems to indication Sonarr is trying to move a file with an incomplete file path.

**Logs**
From the log:
"Import failed, path does not exist or is not accessible by Sonarr: C:\Users\Justin\Downloads\Saturday.Night.Live.S46E06.Dave.Chappelle.720p.WEB.h264-KOGi[TGx"

Notice the truncation at the end.

**System Information**

 - Sonarr Version: 2.0.0.5344
 - Operating System: Windows 10
 - qBittorrent version: 4.3.0.1

**Additional context**
The actual folder name is "Saturday.Night.Live.S46E06.Dave.Chappelle.720p.WEB.h264-KOGi[TGx]" and the main file name inside is "Saturday.Night.Live.S46E06.Dave.Chappelle.720p.WEB.h264-KOGi.mkv" .  Because this is not displayed in the log fully, it makes me think the program is literally trying to move the wrong file path. And as far as I know, this is true, because the moving of the file is failing. I've even set qBittorrent to download files to a different location and the move continues to fail.

I've been using Sonarr for a while and many of the moves are successful so this failure is intermittent. It may have to do with file name length?  **Actually I noticed that the truncated name happens to be the title of the torrent. Is it possible that the Torrent's name is being misused as the folder path during the move operation? 

Indeed! I renamed the title of the torrent in qBitTorrent to be "Saturday.Night.Live.S46E06.Dave.Chappelle.720p.WEB.TorrentName" and now it is showing up in the log as the attempted folder path. This is NOT the path of the torrent files so either the log is wrong or Sonarr is attempting to move a folder that doesn't exist by using the Torrent Name rather than the actual path of the torrent files.**

**Comment by JustinReinhart** [comment]
I've confirmed the bug. If the title of the torrent does not match the folder inside the torrent it results in an automatic failure to move the contents. This is a major bug!

How I confirmed the issue: If I rename the title of the torrent to match the folder contained inside the torrent, Sonarr is able to perform the move operation successfully. 

The fix: In short, Sonarr should not assume the folder inside the torrent is going to match the torrent name.
